### PR TITLE
Support event refinement

### PR DESCRIPTION
**Pull request by fjtirado**
**Description:**
Minor clarification of the level of compliance of our implementation with version 0.8  of the specification, regarding events

Please make sure that your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/getting-started/cncf-serverless-workflow-specification-support.adoc [hunk 1]
```diff
@@ -179,9 +179,10 @@ For additional functions, the Serverless Workflow specification support the `cus
 [[events]]
 == Events
 
-Kogito supports events of the Workflow model as defined in the link:{spec_doc_url}#Event-Definition[Serverless Workflow specification definition]. However, `correlation` events has limited support.
+Kogito supports events of the Workflow model as defined in the link:{spec_doc_url}#Event-Definition[Serverless Workflow specification definition], except:
 
-For more information about the events feature, see xref:eventing/event-correlation-with-workflows.adoc[Event correlation with Serverless Workflows].
+* `consumeEventRef` property in link:{spec_doc_url}#eventref-definition[EventRefDefintion] is not implemented. Same functionality can be achieved by using xref:eventing/working-with-callbacks.adoc[Callback] state.
```
Comment (ricardozanini) [inline serverlessworkflow/modules/ROOT/pages/getting-started/cncf-serverless-workflow-specification-support.adoc:184]:
```suggestion
* `resultEventRef` property in link:{spec_doc_url}#eventref-definition[EventRefDefintion] is not implemented. Same functionality can be achieved by using xref:eventing/working-with-callbacks.adoc[Callback] state. In this state, the property `resultEventRef` is not required on Kogito implementation.
```

Something like this. WDYT?

Comment (fjtirado) [reply serverlessworkflow/modules/ROOT/pages/getting-started/cncf-serverless-workflow-specification-support.adoc]: ok, please check the rewording

**Review by ricardozanini — APPROVED** [review]

**Review by tiagodolphine — APPROVED** [review]
LGTM, I'll update this section along with the correlation guide, to explain the limitation of correlation.

**Review by hmanwani-rh — COMMENTED** [review]
@fjtirado @ricardozanini Please take a look at a couple of suggestions added below.

**Comment by hmanwani-rh** [comment]
@fjtirado @ricardozanini Merging this PR.

**Pull request merged**